public/index.html

```diff
@@ -42,16 +42,36 @@
             "email": "ebechtel@contractor.usgs.gov",
             "affiliation": { "@type": "Organization", "legalName": "U.S. Geological Survey" }
           },
+          { "@type": "Person",
+            "name": "Jessica Driscoll",
+            "email": "jdriscoll@usgs.gov",
+            "affiliation": { "@type": "Organization", "legalName": "U.S. Geological Survey" }
+          },
           { "@type": "Person",
             "name": "John Hammond",
             "email": "jhammond@usgs.gov",
             "affiliation": { "@type": "Organization", "legalName": "U.S. Geological Survey" }
           },
           { "@type": "Person",
-            "name": "Jessica Driscoll",
-            "email": "jdriscoll@usgs.gov",
+            "name": "Graham Sexstone",
+            "email": "sexstone@usgs.gov",
+            "affiliation": { "@type": "Organization", "legalName": "U.S. Geological Survey" }
+          },
+          { "@type": "Person",
+            "name": "Annie Putman",
+            "email": "aputman@usgs.gov",
+            "affiliation": { "@type": "Organization", "legalName": "U.S. Geological Survey" }
+          },
+          { "@type": "Person",
+            "name": "Theodore Barnhart",
+            "email": "tbarnhart@usgs.gov",
+            "affiliation": { "@type": "Organization", "legalName": "U.S. Geological Survey" }
+          },
+          { "@type": "Person",
```

Comment:
We should have Jess as the last author. I think me, you, Marty,others alphabetical, then Jess

Reply:
same in IPDS